config/peerpods/podvm/podvm-builder.sh

```diff
@@ -317,8 +322,8 @@ function delete_podvm_image() {
     if [ "${UPDATE_PEERPODS_CM}" == "yes" ]; then
       kubectl patch configmap peer-pods-cm -n openshift-sandboxed-containers-operator --type merge -p "{\"data\":{\"LIBVIRT_IMAGE_ID\":\"\"}}"
     fi
-
-    ;;
+    
+  ;;
```

Comment:
White space damage... 
```suggestion

  ;;
```

Reply:
Fixed.